dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/DssRestChannel.java

```diff
@@ -18,16 +19,21 @@
  */
 public class DssRestChannel implements DssChannel<DssRestChannelProperty, DssRestChannelInitializer> {
 
+
     @Override
     public Channel bind(ServerBootstrap serverBootstrap, DssRestChannelProperty dssRestChannelProperty, DssRestChannelInitializer dssRestChannelInitializer) throws InterruptedException {
 
         Objects.requireNonNull(serverBootstrap);
         Objects.requireNonNull(dssRestChannelProperty);
         Objects.requireNonNull(dssRestChannelInitializer);
 
+        LogLevel level = LogLevel.DEBUG; //User can change LogLevel
+        String logLevel = level.name();
+        DssLogLevel dssLogLevel = DssLogLevel.valueOf(logLevel);
+
         return serverBootstrap
                 .channel(NioServerSocketChannel.class)
-                .handler(new LoggingHandler(LogLevel.DEBUG))
+                .handler(new LoggingHandler(dssLogLevel.name()))
```

Comment:
Is it safe to converting dss log level to netty log levle?

io.github.ztkmkoo.dss.core.network.rest.enumeration.DssLogLevel has field `private LogLevel level`. Why not make getter like 
``` java
public enum DssLogLevel {

    ERROR(LogLevel.ERROR), WARN(LogLevel.WARN), INFO( LogLevel.INFO), DEBUG( LogLevel.DEBUG), TRACE( LogLevel.TRACE);

    private LogLevel level;

    DssLogLevel( LogLevel level) {
        this.level = level;
    }

    LogLevel getNettyLogLevel() {
        return level;
    }
```

and you can set LoggingHandler like
``` java
@Override
public Channel bind(ServerBootstrap serverBootstrap, DssRestChannelProperty dssRestChannelProperty, DssRestChannelInitializer dssRestChannelInitializer) throws InterruptedException {
    Objects.requireNonNull(serverBootstrap);
    Objects.requireNonNull(dssRestChannelProperty);
    Objects.requireNonNull(dssRestChannelInitializer);
    
    return serverBootstrap
                    .channel(NioServerSocketChannel.class)
                    .handler(new LoggingHandler(dssRestChannelProperty.getDssLogLevel().getNettyLogLevel()))
                    .childHandler(dssRestChannelInitializer)
                    .bind(dssRestChannelProperty.getHost(), dssRestChannelProperty.getPort())
                    .sync()
                    .channel();
}
```